platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,742 @@
+  1. Create a new text file named `sampleinfo.txt`, with two columns:
+      - First column header: Sample
+      - Second column header: Group
+  1. For each sample in your `salmon.merged.gene_counts.tsv` file:
+      - In the "Sample" column, write the exact sample name as it appears in the gene counts file.
+      - In the "Group" column, write the corresponding group name.
+
+  For example, for the dataset used in a `test_full` run of nf-core/rnaseq, the `sampleinfo.txt` looks like this:
+
+  ```
+  Sample        Group
+  GM12878_REP1  GM12878
+  GM12878_REP2  GM12878
+  H1_REP1       H1
+  H1_REP2       H1
+  K562_REP1     K562
+  K562_REP2     K562
+  MCF7_REP1     MCF7
+  MCF7_REP2     MCF7
+  ```
+
+  To make your `sampleinfo.txt` file accessible to the data studio, upload it to the directory that contains your pipeline output data. Select this bucket or directory when you **Mount data** during data studio setup.
+
+</details>
+
+### Create an RStudio analysis environment with Data Studios
+
+![Add data studio](./_images/create-ds.gif)
+
+From the **Data Studios** tab, select **Add a data studio** and complete the following:
+- Select the latest **RStudio** container image template from the dropdown list.
+- Select your AWS Batch compute environment. 
+:::note
+Data studios compete for computing resources when sharing compute environments. Ensure your compute environment has sufficient resources to run both your pipelines and data studio sessions. The default CPU and memory allocation for a data studio is 2 CPUs and 8192 MB RAM. 
+:::
+- Mount data using Data Explorer: Mount the S3 bucket or directory path that contains the pipeline work directory of your RNA-Seq run. 
+- Optional: Enter CPU and memory allocations. The default values are 2 CPUs and 8192 MB memory (RAM).
+- Select **Add**.
+- Once the data studio has been created, select the options menu next to it and select **Start**.
+- When the data studio is in a running state, **Connect** to it. 
+
+### Perform the analysis and explore results
+
+The RStudio environment can be configured with the packages you wish to install and the R script you wish to run. For the purposes of this guide, run the following scripts in the RStudio console to install the necessary packages and perform the analysis:
+
+1. Install and load the necessary packages and libraries:
+
+    ```console
+    # Install required packages
+    if (!requireNamespace("BiocManager", quietly = TRUE))
+      install.packages("BiocManager")
+    BiocManager::install(c("limma", "edgeR", "ggplot2", "gplots"))
+
+    # Load required libraries
+    library(limma)
+    library(edgeR)
+    library(ggplot2)
+    library(gplots)
+    ```
+
+1. Read and convert the count data and sample information:
+
+    :::info
+    Replace `<PATH_TO_YOUR_COUNTS_FILE>` and `<PATH_TO_YOUR_SAMPLE_INFO_FILE>` with the paths to your `salmon.merged.gene_counts.tsv` and `sampleinfo.txt` files.
```

Comment:
I'm not sure how pedantic we want to be here about correctness in analysis.

There can be important differences in effective gene length across conditions, which we normally account for in analysis. Using salmon.merged.gene_counts.tsv will just ignore that effect.

We can either model those length differences (preferable, but would probably add some unnecessary complexity here), or otherwise just use salmon.merged.gene_counts_length_scaled.tsv (which is probably the simplest thing to do here).

Reply:
The workflow does give you the raw outputs from Salmon / Kallisto, which are at the transcript level, and which is what tximport reads.

But it also uses tximport internally to produce those count matrices (salmon.merged.gene_counts_length_scaled.tsv, salmon.merged.gene_counts.tsv), and provides gene lengths (salmon.merged.gene_lengths.tsv) that can be used as offsets directly in downstream analysis.

Reply:
I'd err on the side of pedantic. So need to run the script again with salmon.merged.gene_counts_length_scaled.tsv and update the GIFs and steps. Thanks for the detailed feedback gents!